src/fidesops/cli.py

```diff
@@ -18,3 +19,13 @@ def webserver(ctx: click.Context) -> None:
     Runs any pending DB migrations and starts the webserver.
     """
     start_webserver()
+
+
+@cli.command()
+@click.pass_context
+def worker(ctx: click.Context) -> None:
+    """
+    Starts a Celery worker
+    """
+    celery_args = ["worker", "--loglevel=info"]
```

Comment:
This now becomes the place to [pass through args to celery workers](https://docs.celeryq.dev/en/stable/userguide/workers.html), like `--loglevel` and `--concurrency`, if we wanted to make them config vars.